### ISSUE TITLE
Problem with dependencies for ubuntu (low priority)

### ISSUE THREAD

**Issue by iantaylor-NOAA** [pfmc-assessments/PacFIN.Utilities#91]
tl;dr: dependencies on linux aren't installing right and it's seems to be not our fault so we should ignore it for now.

**Describe the bug**
I looked deeper at the recent github actions that are failing on ubuntu.
They are failing on the step where the R dependencies are installed (`Run r-lib/actions/setup-r-dependencies@v2`) and the first error (on this [line](https://github.com/pfmc-assessments/PacFIN.Utilities/actions/runs/4186350745/jobs/7254729842#step:5:4463)) is
```
dpkg: error processing archive /var/cache/apt/archives/libodbc1_2.3.11_amd64.deb (--unpack):
   trying to overwrite '/usr/lib/x86_64-linux-gnu/libodbc.so.2.0.0', which is also in package libodbc2:amd64 2.3.9-5
  dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
```

A similar error was described 4 days ago here: https://askubuntu.com/questions/1454448/trying-to-overwrite-foo-which-is-also-in-package-bar-dpkg-deb-error-paste-sub where the second answer seems most useful to us:

> Your ouput says that you have two packages, libodbc1 and libodbc2 that are trying to provide the same file (libodbc.so.2.0.0). That's bad.
>
> In other words, those two packages are incompatible. Only one of them can be installed without forcing (--force-overwrite). Only an experienced user (like you) should hazard forcing, and record somewhere that you did so -- uninstalling A may require reinstalling B.
>
> Inexperienced users should not try to install both using deb package. Installing one from source, or using a different package system, may be preferable.

I presume that something changed in these packages recently as our our GitHub actions were doing fine until yesterday.
I'm guessing that nobody is running PacFIN.Utilities on linux so we can safely ignore this and trust that somebody will fix it. If someone is more enthusiastic than me, I guess we could report this to the developers of RODBC or whichever of the PacFIN.Utilities dependencies is requiring these libraries to be installed.

**To Reproduce**
1. Push changes to the PacFIN.Utilities repo
2. Wait for the [github actions](https://github.com/pfmc-assessments/PacFIN.Utilities/actions/) to run
3. Look for the error messages in the failed ubuntu action, like this one.

**Expected behavior**
Successful GitHub actions

**Comment by kellijohnson-NOAA** [comment]
I believe this fixed itself 🥳
